CONTRIBUTING.md

```diff
@@ -20,56 +125,95 @@ of this plugin:
+
+    ```sh-session
+    docker-compose up -d conjur-cli
+    ```
+
+## Load policy to set up Conjur Ansible integration
+
+  Policy defines Conjur entities and the relationships between them.  An entity can be a policy, a host, a user, a layer, a group, or a variable.
+
+  Check out the policy file, and load it into Conjur:
 
   ```sh-session
-   $ cd dev
-   $ ./start.sh
-   ...
-   root@f75015718049:/cyberark/dev/#
+  docker-compose exec conjur_cli cat /policy/root.yml
+  docker-compose exec conjur_cli conjur policy load root /policy/root.yml
   ```
 
-   Once the `start` script finishes, you're in a Bash shell inside the Conjur
-   server container.  To
+  Also, load a dummy secret value into the 'ansible/target-password' variable. This is a variable required by remote nodes in order to complete their workloads.
 
-   After starting Conjur, your instance will be configured with the following:
-   * Account: `cucumber`
-   * User: `admin`
-   * Password: Run `cat conjur.identity` inside the container shell to display the current logged-in identity (which is also the password)
+  ```sh-session
+  docker-compose exec conjur_cli conjur variable values add ansible/target-password S3cretV@lue
+  ```
+## Create Ansible managed nodes
 
-1. Debug the server
+  The Ansible environment will include a control node and a number of managed nodes. First, retrieve the API key for the Conjur host representing the control node, then create it:
 
   ```sh-session
-   root@f39015718062:docker-compose exec <container-service-name> bash
-   <various startup messages, then finally:>
-   Use exit to stop
-   ```
+  export ANSIBLE_CONJUR_AUTHN_API_KEY="$(docker-compose exec conjur conjurctl role retrieve-key cucumber:host:ansible/ansible-master)"
+  docker-compose up -d ansible
+  ```
 
-## Testing
+  Next, create two instances of each managed node:
 
-    To run a specific set of tests:
+  ```sh-session
+  docker-compose up -d --scale test_app_ubuntu=2 test_app_ubuntu
+  docker-compose up -d --scale test_app_centos=2 test_app_centos
+  ```
 
-    ```sh-session
-     $ cd tests
-     $ ./test.sh -d <role or plugin name>
-     ```
-    To run all tests:
-
-     ```sh-session
-     $ cd tests
-     $ ./test.sh -a
-     ```
+## Use conjur host identity to set up Conjur identity on managed nodes
+
+  To grant your Ansible host a Conjur identity, you first must install the Conjur Ansible Role in your playbook directly
+
+  ```sh-session
+  ansible-galaxy install cyberark.conjur-host-identity
+  ```
+  Once you've done this, you can configure each Ansible node with a Conjur identity by including a section like the example below in your Ansible playbook:
+
+  ```sh-session
+  - hosts: servers
+    roles:
+      - role: cyberark.conjur-host-identity
+        conjur_appliance_url: 'https://conjur.myorg.com',
+        conjur_account: 'myorg',
+        conjur_host_factory_token: "{{lookup('env', 'HFTOKEN')}}",
```

Comment:
We need an additional step before this one setting up the host factory token in the `HFTOKEN` env var

Reply:
Done